deploy/olm-catalog/ibm-mongodb-operator/0.0.1/ibm-mongodb-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -42,37 +42,15 @@ spec:
   customresourcedefinitions:
     owned:
     - description: MongoDB is the Schema for the mongodbs API
-      displayName: MongoDB
```

Comment:
We need to keep `displayName:` and `resources:` fields. They are necessary for CSV.